src/Asiago/Commands/TwitchModule.cs

```diff
@@ -0,0 +1,219 @@
+                        // then just delete the twitch channel and let things cascade
+                        if (twitchChannel.SubscribedGuilds.Count == 1)
+                        {
+                            bool subscriptionDeleted = await _twitchApi.Helix.EventSub.DeleteEventSubSubscriptionAsync(twitchChannel.SubscriptionId);
+                            if (!subscriptionDeleted)
+                            {
+                                await ctx.RespondAsync("Something went wrong...");
+                                return;
+                            }
+                            dbContext.Remove(twitchChannel);
+                        }
+                        else
+                        {
+                            twitchChannel.SubscribedGuilds.Remove(guild);
+                        }
+                        try
+                        {
+                            await dbContext.SaveChangesAsync();
+                            await ctx.RespondAsync($"Removed Twitch channel [{twitchChannelName}] subscription for this guild");
+                            return;
+                        }
+                        catch (DbUpdateException ex)
+                        {
+                            // This can happen due to a race condition due to the delay between looking if an item exists in the db
+                            // and removing it if it does. This really shouldn't happen often.
+                            await ctx.RespondAsync("Something went wrong...");
+                            _logger.LogWarning(
+                                ex,
+                                "Failed to remove subscription to Twitch channel [{twitchUserId}][{twitchChannelName}] for guild [{guildId}][{guildName}]",
+                                twitchUser.Id,
+                                twitchChannelName,
+                                ctx.Guild.Id,
+                                ctx.Guild.Name
+                                );
+                            return;
+                        }
+                    }
+                }
+            }
+
+            await ctx.RespondAsync($"Not subscribed to Twitch channel [{twitchChannelName}] in this guild");
+        }
+
+        [Command]
+        public async Task List(CommandContext ctx)
+        {
+            List<string> twitchUserIds;
+            using (var dbContext = _dbContextFactory.CreateDbContext())
+            {
+                twitchUserIds = await dbContext.TwitchChannels
+                    .Where(tc => tc.SubscribedGuilds.Any(g => g.GuildId == ctx.Guild.Id))
+                    .Select(tc => tc.UserId)
+                    .ToListAsync();
+            }
+
+            if (twitchUserIds.Count > 0)
+            {
+                var twitchGetUsersResponse = await _twitchApi.Helix.Users.GetUsersAsync(twitchUserIds);
+                if (twitchGetUsersResponse.Users.Count() > 0)

```

Comment:
switch around so the empty case is in a block and the other case is below. add a "something went wrong" message and and error log for the empty case.